content-api-crud/search.yml

```diff
@@ -0,0 +1,44 @@
+execution:
+- concurrency: 1
```

Comment:
Would it be possible to configure concurrency via the command line, or do you need to update it in the yml?

Reply:
yes, that is documented in the existing readme.